x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/hooks/use_endpoint_action_items.tsx

```diff
@@ -100,7 +101,7 @@ export const useEndpointActionItems = (
             />
           ),
         });
-      } else if (isPlatinumPlus && isolationSupported) {
+      } else if (isPlatinumPlus && isolationSupported && canIsolateHost) {
```

Comment:
I think the license check here should be removed, since `authz` already checks it when setting the `canIsolateHost` property.